Update PULL_REQUEST_TEMPLATE.md (#1067)
cosmos/ibc-rs: RELEASES.md

    then update the release description and append:
   `[📖CHANGELOG](https://github.com/cosmos/ibc-rs/blob/main/CHANGELOG.md#vXYZ)`

> Communications (non-technical) release pipeline:

Reviewer: ```suggestion
### Communications (non-technical) release pipeline
```

> - Notify the communications team about the pending release and prepare a message to announce it.
> - Coordinate with other organizations that are active in IBC development (e.g., Interchain) to ensure they are in the loop.
> 

Reviewer: ```suggestion
- Notify the communications team about the pending release and prepare an announcement.
- Coordinate with other organizations that are active in IBC development (e.g., Interchain) and keep them in the loop.

```
